heroku/cli
# refactor: migrate apps:lock to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001awBu4YAE/view)

## Note
We cannot delete the orgs-v5/commands/apps/lock file until we convert the apps:transfer command.

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Choose an unlocked app (I used k80-test)
- Run `./bin/run apps:lock -a APP-NAME`
    - You should get a message that the app has been locked
- Run the same command again
    - You should get an error message that the app has already been locked
- Run `heroku apps:unlock -a APP-NAME` to reset
